libs/blocks/preflight/panels/assets.js

```diff
@@ -0,0 +1,188 @@
+import { html, signal, useEffect } from '../../../deps/htm-preact.js';
+import { createTag } from '../../../utils/utils.js';
+
+function isViewportTooSmall() {
```

Comment:
why not
```js
const isViewportTooSmall = () => !window.matchMedia('(min-width: 1200px)').matches;
```

Reply:
This is mostly because the Preflight logic in general uses `function` definitions, I wanted to stick to that pattern.